oidc-client.cabal

```
@@ -46,7 +46,7 @@ library
       Web.OIDC.Client.Internal
   build-depends:
       base              >=4.7 && <5
-    , bytestring        >=0.10 && <0.11
+    , bytestring        >=0.11
```

Is it possible to set `>=0.10`? Because resolving dependencies failed in TLS 16.31.
```suggestion
    , bytestring        >=0.10
```